src/utils/viewer/simviewer.cpp

```diff
@@ -0,0 +1,724 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <stdlib.h>
+
+#include <Magnum/configure.h>
+#include <Magnum/ImGuiIntegration/Context.hpp>
+#ifdef MAGNUM_TARGET_WEBGL
+#include <Magnum/Platform/EmscriptenApplication.h>
```

Comment:
Looks like one of the other includes you added broke the emscripten build. Hmm...